src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -175,18 +175,23 @@ public void onActivityCreated(Activity activity, Bundle bundle) {
 
         @Override
         public void onActivityStarted(Activity activity) {
+            ConnectivityIndicator.start(mContext);
```

Comment:
Again, we should only initialize it if we have to. You created a dependency on the singleton pattern which makes it not singleton basically by having to call `start` before `show` every time.